foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,44 +1,80 @@
+
+1. Create a new function named `playRound`.
+1. Define two parameters for `playRound`: `playerChoice` and `computerChoice`. Use these two parameters to take the human and computer choices as arguments.
+1. Make your function's `playerSelection` parameter case-insensitive so that players can input "rock", "ROCK", "RocK", or other variations.
+1. Write the code for your `playRound` function to `return` a string value representing the round winner, such as: "You lose! Paper beats Rock".
+
+Example code:
+
+```javascript
+function playRound(playerChoice, computerChoice) {
+  // your code here!
+}
+
+const playerSelection = "rock";
+const computerSelection = getComputerChoice();
 
-1. Start a new Git repo for your project.
-1. Create a blank HTML document with a script tag (Hint: it is best practice to link an external .js file). This game is going to be played completely from the console, so don't worry about putting anything else in there.
-1. Your game is going to play against the computer, so begin with a function called `getComputerChoice` that will randomly return either 'Rock', 'Paper' or 'Scissors'. We'll use this function in the game to make the computer's play. *Tip: use the console to make sure this is returning the expected output before moving to the next step!*
-1. Write a function that plays a single round of Rock Paper Scissors. The function should take two parameters - the `playerSelection` and `computerSelection` - and then return a string that declares the winner or tie of the round like so: `"You Lose! Paper beats Rock"`
+console.log(playRound(playerSelection, computerSelection));
+```
 
-   - Make your function's playerSelection parameter case-insensitive (so users can input `rock`, `ROCK`, `RocK` or any other variation).
+#### Step 4: Play game function
 
-1. **Important note:** you want to `return` the results of this function call, *not* `console.log()` them. You're going to use what you `return` later on, so let's test this function by using console.log to see the results:
+Your game will play 5 rounds. Write a function named `playGame` that uses `playRound` to play 5 rounds, keeps score, and declares a winner at the end:
 
-   ```javascript
-   function playRound(playerSelection, computerSelection) {
-     // your code here!
-   }
+1. Create a new function named `playGame`.
+1. Get the human player choice using the `prompt` method. Read this [MDN article about the prompt method](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt) learn more about it.
+1. Put the previous `playRound` function inside `playGame`.
+1. Play 5 rounds by calling `playRound` 5 times.
+   - **Hint**: When you assign a function call to a variable, the return value of that function is assigned to the variable. Accessing the variable afterward will only provide the assigned value; it doesn't recall the function. You need to recall the choice functions to get new choices for each round.
```

Comment:
If we're keeping the verbiage about loops it would make more sense to include it here

```suggestion
   - **Hint**: When you assign a function call to a variable, the return value of that function is assigned to the variable. Accessing the variable afterward will only provide the assigned value; it doesn't recall the function. You need to recall the choice functions to get new choices for each round.
   - If you already know about loops, you can use them. If not, don't worry! Loops will be covered in the next lesson.
```